test/connectivity.jl

```diff
@@ -111,12 +127,20 @@
 
     for g in testdigraphs(h)
       scc = @inferred(strongly_connected_components(g))
+      scc_k = @inferred(strongly_connected_components_kosaraju(g))
       @test length(scc) == 4
       @test sort(scc[1]) == [7, 8, 9, 10, 11, 12]
       @test sort(scc[2]) == [3, 6]
       @test sort(scc[3]) == [2, 4, 5]
       @test scc[4] == [1]
+        
+      @test length(scc_k) == 4
+      @test sort(scc_k[1]) ==  [1] 
+      @test sort(scc_k[2]) ==  [2, 4 ,5] 
+      @test sort(scc_k[3]) ==  [3, 6]  
+      @test scc_k[4] ==  [11, 12, 10, 9, 7, 8]
```

Comment:
You should also put `sort` here. The reason that `sort`  was not used for `scc` was, that that component only had one element.

Reply:
Also, does kosaraju guarantee that the components are sorted in increasing order of length? Otherwise we should sort the components by length first, before doing these tests.

Reply:
Rewritten to  `sort(scc_k[4])` . No, neither Tarjan nor Kosaraju guarantee that components are sorted in increasing order of length. But why do we need to sort by length first ? The current written tests don't do this.